flexmeasures/ui/utils/view_utils.py

```diff
@@ -91,6 +92,10 @@ def render_flexmeasures_template(html_filename: str, **variables):
     if "asset" in variables:
         variables["breadcrumb_info"] = get_breadcrumb_info(asset)
 
+    # add color settings
+    account = db.session.query(Account).get(current_user.account_id)
```

Comment:
You should be able to use `current_user.account`, which is shorter and maybe faster.

Reply:
Yes, that makes sense